test/envs/demo.yaml

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Example environment.
+---
+name: demo
+templates:
+  - name: "demo-cluster"
+    driver: kvm2
+    container_runtime: containerd
+    network: default
+    addons:
+      - ingress
+    workers:
+      - scripts:
```

Comment:
This was renamed to `addons` - see other environment files.

This `scripts` key is ignored by drenv, so this environment actually runs nothing when it starts:

```
$ drenv dump envs/demo.yaml 
name: demo
profiles:
- addons:
  - ingress
  cni: auto
  container_runtime: containerd
  cpus: 2
  disk_size: 20g
  driver: kvm2
  external: false
  extra_disks: 0
  memory: 4g
  name: demo1
  network: default
  nodes: 1
  workers:
  - addons: []    <<<< empty addons list (default)
    name: demo1/0
    scripts:      <<<< this is ignored
    - name: demo
...
```

When we start nothing is deployed:

```
$ kubectl get deploy -A --context demo1
NAMESPACE       NAME                       READY   UP-TO-DATE   AVAILABLE   AGE
ingress-nginx   ingress-nginx-controller   1/1     1            1           7m15s
kube-system     coredns                    1/1     1            1           7m17s
```

This works for me:

```
--- a/test/envs/demo.yaml
+++ b/test/envs/demo.yaml
@@ -12,7 +12,7 @@ templates:
     addons:
       - ingress
     workers:
-      - scripts:
+      - addons:
           - name: demo
 profiles:
   - name: demo1
```

Of course "works" means that drenv start fails:

```
$ drenv start envs/demo.yaml 
Traceback (most recent call last):
  File "/home/nsoffer/.venv/ramen/bin/drenv", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/nsoffer/src/ramen/test/drenv/__main__.py", line 44, in main
    env = envfile.load(f, name_prefix=args.name_prefix)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nsoffer/src/ramen/test/drenv/envfile.py", line 17, in load
    _validate_env(env, addons_root)
  File "/home/nsoffer/src/ramen/test/drenv/envfile.py", line 40, in _validate_env
    _validate_profile(profile, addons_root)
  File "/home/nsoffer/src/ramen/test/drenv/envfile.py", line 91, in _validate_profile
    _validate_worker(worker, profile, addons_root, i)
  File "/home/nsoffer/src/ramen/test/drenv/envfile.py", line 99, in _validate_worker
    _validate_addon(addon, env, addons_root, args=[env["name"]])
  File "/home/nsoffer/src/ramen/test/drenv/envfile.py", line 108, in _validate_addon
    raise MissingAddon(addon["name"])
drenv.envfile.MissingAddon: demo
```

Moving the `test/demo` directory to `test/addons/demo` will fix this.